test/functionalTest/cases/0501_cors/specific_origin.test

```diff
@@ -0,0 +1,156 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+CORS specific origin
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0-255 IPV4 -corsOrigin arrakis
+
+--SHELL--
+echo "01. GET Entry Points (Access-Control-Allow-Origin header not included)"
+echo "======================================================================"
+orionCurl --url /v2
+echo
+echo
+
+echo "02. GET Entry Points with arrakis origin (Access-Control-Allow-Origin header included)"
+echo "======================================================================================"
+orionCurl --url /v2 -X GET --origin arrakis
+echo
+echo
+
+echo "03. GET Entry Points with caladan origin (Access-Control-Allow-Origin header not included)"
+echo "=========================================================================================="
+orionCurl --url /v2 -X GET --origin caladan
+echo
+echo
+
+echo "04. OPTIONS Entry Points (Access-Control-Allow-Origin header not included)"
+echo "=========================================================================="
+orionCurl --url /v2 -X OPTIONS
+echo
+echo
+
+echo "05. OPTIONS Entry Points with arrakis origin (Access-Control-Allow-Origin header included)"
+echo "=========================================================================================="
+orionCurl --url /v2 -X OPTIONS --origin arrakis
```

Comment:
It would be great to include a .test checking that when brokerStart doesn't use `-corsOrigin` then OPTIONS /v2 (or OPTSION /v2/entities) results in 405 Method not allowed.

This could be include in this PR (in this case, I'll wait for your "Fixme in") or in a next one (in this case, I'll wait for your NTC), as you prefer.

Reply:
Fixed in f97a6f1